contexts/credentials/v2

```diff
@@ -66,6 +66,14 @@
       "@id": "https://www.iana.org/assignments/jwt#..."
     },
 
+    "digestSRI": {
+      "@id": "https://www.w3.org/2018/credentials#digestSRI",
+      "@type": "https://www.w3.org/2018/credentials#sriString"
+    },
+    "mediaType": {
+      "@id": "https://schema.org/encodingFormat"
+    },
+
```

Comment:
Hmm, do we want both of these as top-level values... or do we want them under `relatedResource`. I would imagine we'd want the latter?

Reply:
@msporny I am open either way, I leave the decision to you guys.

My argument for this setup is that `digestSRI` might be useful elsewhere; there is nothing in the definition that is really bound to `relatedResource`. After all, this is a general mechanism. (Note that the domain of `digestSRI` is essentially open-ended, see also https://github.com/w3c/vc-data-model/pull/1296#issuecomment-1742010597). But, again, I regard this as a VCDM domain issue, and I am not a VCDM expert, so to say...